app/utils/buttonStyles.ts

```diff
@@ -104,15 +105,23 @@ export const buttonBackgroundStyle = (
             disabled: {
                 default: {
                     backgroundColor: changeOpacity(theme.centerChannelColor, 0.08),
+                    borderColor: changeOpacity(theme.centerChannelColor, 0.32),
```

Comment:
I couldn't add a comment on line 68, but I believe primary buttons do not have a border even when they are focused, is that right @matthewbirtch ?

Reply:
Whoops no - you're right; I think this was a copy/paste fail - I'll fix up.

All the buttons are here: https://www.figma.com/file/2uYWxjnMJ9IQDOba9b9bfV/%E2%9C%85Components---Buttons?node-id=1%3A184